.github/workflows/Image.yaml

```diff
@@ -262,42 +278,42 @@ jobs:
           IS_A_ROCK: ${{ matrix.dockerfile-build == '' && true || false }}
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          ref: ${{ github.ref_name }}
           fail-on-error: true
           workflow-name: Tests.yaml
           payload: '{ "oci-image-name": "${{ steps.rename-oci-archive.outputs.name }}", "is-a-rock": ${{ env.IS_A_ROCK }}, "cache-key": "${{ github.run_id }}-${{ steps.rename-oci-archive.outputs.name }}", "vulnerability-report-suffix": "${{ env.VULNERABILITY_REPORT_SUFFIX }}" }'
-          use-marker-step: true
-          trigger-timeout: '3m'
-
-      - name: Wait for tests
-        uses: mathze/workflow-dispatch-action@v1.2.0
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          run-id: ${{ steps.run-tests.outputs.run-id }}
-          wait-interval: 10s
+          trigger-timeout: "3m"
+          run-id: dummy
       
+      - name: Write step summary
+        run: |
+          url='${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ steps.run-tests.outputs.run-id }}'
+          echo " - Triggered tests for '${{ matrix.name }}' at [${url}](${url})" >> "$GITHUB_STEP_SUMMARY"
+
+      # Wait to make sure the test artifacts are uploaded
+      - run: sleep 5
+
       - name: Download vulnerability artifacts
         uses: dawidd6/action-download-artifact@v2
         with:
           workflow: Tests.yaml
           run_id: ${{ steps.run-tests.outputs.run-id }}
-          path: '.'
+          path: "."
           name: ${{ steps.rename-oci-archive.outputs.name }}${{ env.VULNERABILITY_REPORT_SUFFIX }}.zip
 
-      - run: ls *
-      
       - name: Install Syft
         uses: anchore/sbom-action/download-syft@v0
         with:
-          syft-version: 'v0.75.0'
-    
```

Comment:
sorry, I don't understand the question. We are versioning Syft and yq. What's the issue?

Reply:
I was wondering if we can utilize the latest version of Syft?

Reply:
any specific reason why? pinpointing is a good practice cause otherwise you might unintentionally get new releases that could be backward incompatible or unstable

Reply:
Yes, you're right. I didn't think about it.